<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridWidget</class>
 <widget class="QWidget" name="GridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="sortLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sort by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Divisions</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Users</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchField">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectAll">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select all</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/select</normaloff>:/icons/select</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unselectAll">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Unselect all</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/unselect</normaloff>:/icons/unselect</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showSelected">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Show selected</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/selected</normaloff>:/icons/selected</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>572</width>
        <height>292</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sortBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GridWidget</receiver>
   <slot>sortBy(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GridWidget</receiver>
   <slot>searchCountry(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAll</sender>
   <signal>clicked()</signal>
   <receiver>GridWidget</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unselectAll</sender>
   <signal>clicked()</signal>
   <receiver>GridWidget</receiver>
   <slot>unselectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSelected</sender>
   <signal>clicked()</signal>
   <receiver>GridWidget</receiver>
   <slot>showSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sortBy(int)</slot>
  <slot>searchCountry(QString)</slot>
  <slot>selectAll()</slot>
  <slot>unselectAll()</slot>
  <slot>showSelected()</slot>
 </slots>
</ui>
